test/components/util/util.go

```diff
@@ -164,7 +164,9 @@ func WaitForDaemonSet(t *testing.T, client kubernetes.Interface, ns, name string
 				return false, nil
 			}
 
-			return false, fmt.Errorf("getting DaemonSet %q: %w", name, err)
+			t.Logf("getting DaemonSet %q: %v", name, err)
```

Comment:
Depending on the main-retry-loop to retry can elongate the time for test if there actually is error and not something transient. We might think about introducing shorter retry for such stuff. This does not have to be here, could reside outside as a helper func.

Reply:
Right. But on the other hand, it also gives more time to investigate when you see that it loops here.